src/main/java/seedu/address/logic/commands/FindTaskCommand.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Finds and lists all tasks in the task list that are related to a specified person.
+ */
+public class FindTaskCommand extends Command {
+
+    public static final String COMMAND_WORD = "findtask";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all tasks related to the person identified by "
```

Comment:
I like how you structured the detailed command usage. This is very efficient in  providing clear instructions on how to use the command, which greatly aids in user understanding and reduces user errors.